code/modules/client/preference_setup/background/03_records.dm

```diff
@@ -4,7 +4,6 @@
 	var/sec_record = ""
 	var/gen_record = ""
 	var/memory = ""
-	var/email_addr = ""
```

Comment:
Why are we fucking with emails here?

Reply:
E-mail domains are set by branches... which were removed. Ergo, e-mail code was simplified. There was no way to have a domain other than `site53.foundation` anyways.

Reply:
aight its fine then